Rakefile

```diff
@@ -1,6 +1,12 @@
 # encoding: utf-8
 
-require 'rubygems'
+begin
+  require 'bundler/setup'
+  require 'bundler/gem_tasks'
```

Comment:
AFAIK this is the only one we need, looking at the Rakefile from a freshly generated gem. Template here: https://github.com/bundler/bundler/blob/master/lib/bundler/templates/newgem/Rakefile.tt

Reply:
`bundler/setup` lets you skip the whole `Bundler.setup` block but since we already have that, just removed it. Should be good now